kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/crds/Dummy.java

```diff
@@ -20,10 +20,13 @@
 import io.fabric8.kubernetes.client.CustomResource;
 import io.fabric8.kubernetes.model.annotation.Group;
 import io.fabric8.kubernetes.model.annotation.Version;
+import io.fabric8.kubernetes.model.annotation.Plural;
 
 @Version(Dummy.VERSION)
 @Group(Dummy.GROUP)
+@Plural(Dummy.PLURAL)
```

Comment:
Is that needed?

Reply:
I was using `Utils.getPluralFromKind` for resolving plural which wasn't handling it. I've changed informers to use `Pluralize.toPlural` instead which now handle this case

Reply:
The problem is that this logic should be taken from `CustomResource` so that we don't have several different implementations that more or less do the same thing albeit sightly differently.  Similarly, there should be only one method that does the pluralization and everything should use it instead of having 2 different methods.